compare_2_files/document_similarity_using_words_comparision.py

```diff
@@ -0,0 +1,183 @@
+            "which",
+            "like",
+            "been",
+            "in",
+            "or",
+            "she",
+            "him" "call",
+            "is",
+            "one",
+            "do",
+            "into",
+            "who",
+            "you",
+            "had" "how",
+            "time",
+            "oil",
+            "that",
+            "by",
+            "their",
+            "has",
+            "its",
+            "it",
+            "word",
+            "if",
+            "look",
+            "now",
+            "he",
+            "but",
+            "will",
+            "two",
+            "find",
+            "was",
+            "not",
+            "up",
+            "more",
+            "long",
+            "for",
+            "what",
+            "other",
+            "write",
+            "down",
+            "on",
+            "all",
+            "about",
+            "go",
+            "day",
+            "are",
+            "were",
+            "out",
+            "see",
+            "did",
+            "as",
+            "we",
+            "many",
+            "number",
+            "get",
+            "with",
+            "when",
+            "then",
+            "no",
+            "come",
+            "his",
+            "your",
+            "them",
+            "way",
+            "made",
+            "they",
+            "can",
+            "these",
+            "could",
+            "may",
+            "I",
+            "said",
+            "so",
+            "people",
+            "part",
+        ]
+        return words_list
+
+    def words(self, add_file1):
+        """
+		This function return the words present in file1 and file 2 with word counter in a list form		
+		"""
+        file1 = open(add_file1, "r")
+        words_from_file = file1.read().split()
+        file1.close()
+
+        return words_from_file
+
+    def common_word_list_ret(self, Comp_list):
+        """
+		This function removes all the common words from list and returns a new list	
+		"""
+        common_word_list = self.common_word_list()
+        new_list = []
+        for word in Comp_list:
+            if word.lower() not in common_word_list:
+                new_list.append(word)
+        return new_list
+
+    def diff_list(self, list1, list2):
+        """
+		This function removes all the similar words from both list and returns a new list	
+		"""
+        final_list = []
+        final_list1 = []
+        final_list2 = []
+        temp_list = []
+        if len(list1) > len(list2):
+            final_list2 = list1
+            final_list1 = list2
+        else:
+            final_list2 = list2
+            final_list1 = list1
+        for words in final_list2:
+            temp_list.append(words.lower())
+        for word in final_list1:
+            if word.lower() not in temp_list:
+                final_list.append(word)
+        return final_list
+
+    def percentile(self, new_list1, new_list2, final_list):
```

Comment:
Type hints on the return value?  This should be a function, not a method because it never uses __self__.